src/OidcClientSettings.ts

```diff
@@ -168,6 +173,7 @@ export class OidcClientSettingsStore {
 
     public readonly revokeTokenAdditionalContentTypes?: string[];
     public readonly fetchRequestCredentials: RequestCredentials;
+    public readonly refreshTokenAllowableScopes: string | undefined;
```

Comment:
Please use type `?: string[];`

Reply:
Ah i see you made it the same as `scope?: string`, i guess, makes sense to keep it like you have